packages/teleport-plugin-angular-base-component/src/utils.ts

```diff
@@ -16,11 +13,7 @@ export const generateComponentDecorator = (uidl: ComponentUIDL, t = types) => {
       t.objectProperty(t.identifier('selector'), t.stringLiteral('app-root')),
       t.objectProperty(
         t.identifier('templateUrl'),
-        t.stringLiteral(`./${getComponentFileName(uidl)}.${FILE_TYPE.HTML}`)
-      ),
-      t.objectProperty(
```

Comment:
Are we going to handle path in project generators alex ?

Reply:
no, I moved this to the css-plugin

Reply:
But the output file is not referring I think

```ts
import { Component } from '@angular/core'

@Component({
  selector: 'app-root',
  templateUrl: 'contact-form.html',
  styleUrls: ['contact-form.css'],
})
export class ContactForm {
  constructor() {}
}
```

Reply:
And if the css is moved, the template should still follow the same for `tempalteUrl` right --> ``templateUrl: './contact-form.html'``